### ISSUE TITLE
mvn generate fails if there are spaces in company name 

### ISSUE THREAD

**Issue by lbruby** [christian-schlichtherle/truelicense#6]
Hi,

Just starting with truelicense. When running the mvn generate as shown on this page, if the company name specified for -Dcompany has any spaces in it, the build fails with an error that looks like what I've pasted in below.  This happens anytime there are spaces in the company name provided for the -Dcompany option, regardless of what the name is.  If I put an _ (underscore) in place of any spaces in the company name the build completes successfully. Are you aware of this issue?

[ERROR] Unknown lifecycle phase "Inc.?". You must specify a valid lifecycle phase or a goal in the format <plugin-prefix>:<goal> or <plugin-group-id>:<plugin-artifact-id>[:<plugin-version>]:<goal>. Available lifecycle phases are: validate, initialize, generate-sources, process-sources, generate-resources, process-resources, compile, process-classes, generate-test-sources, process-test-sources, gen
erate-test-resources, process-test-resources, test-compile, process-test-classes, test, prepare-package, package, pre-integration-test, integration-test, post-integration-test, verify, install, deploy, pre-clean, clean, post-clean, pre-site, site, post-site, site-deploy. -> [Help 1]

Thank you.

Lisa

**Comment by lbruby** [comment]
Update on this. I believe that any spaces in any of the parameters when generating the project are a problem. I am using maven 3.6.3.

**Comment by christian-schlichtherle** [comment]
BTW: This issue should be reported against the `truelicense-maven-archetype` repository.

The example in the `README.md` of this project has a company name with a space in it (`Company Inc.`) and it's working just fine. In fact, all the integration tests of this projects all use this property value.

Please check your property value quoting. The example provided assumes the Bourne Shell. Maybe you need to adapt it for your case.

**Comment by lbruby** [comment]
Thank you for the response. I am working on Windows 8.1 from the command 'shell'. I've tried every method of quoting. I have found a workaround. If I don't provide the properties on the command line and instead let maven prompt me for any values that require spaces, then it works.  I modified the archetype-metadata.xml file in the truelicense-maven-archetype-4.0.2.jar file so it would not have a default for the editions property, so it would prompt for that one too.  I will also try using cygwin with the bash shell to see how that works. Closing this. Apologies for reporting this in the wrong repository. Took me awhile to understand everything. I'm new to maven.
